scripts/docker-entrypoint.sh

```diff
@@ -297,8 +297,8 @@ if [ -e "$context_xml" ]; then
   cp "$context_xml" "${CATALINA_HOME}"/conf/
 else
     if [ "$SECRET_DB_USERNAME" == "" ] ; then
-      echo "DB_USERNAME must be specified.";
-      exit 1
+      echo "DB_USERNAME must be specified. Assuming this as a passwordless connection.";
```

Comment:
This message is little confusing.
This can be written as,

`As DB_USERNAME is not specified, trying password less connection`